libraries/engage/components/MessageBar/style.js

```diff
@@ -45,10 +45,27 @@ const srOnly = css({
   border: 0,
 });
 
+const withIcon = css({
```

Comment:
We should change these files to have named exports. This is better for tree shaking and modularisation.